.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+Thank you for contributing to JanusGraph.
+
+In order to streamline the review of the contribution we ask you
+to ensure the following steps have been taken:
+
+### For all changes:
+- [ ] Is there an issue associated with this PR? Is it referenced 
+     in the commit message?
+
```

Comment:
A line should end on a line feed. Are you sure you want it dropped? A few tools work a tiny bit nicer when there is a new line at the end of the file, e.g. no git "No newline at end of file" message in diffs, adding a line won't change the last line, etc.